templates/new/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%= appName %>",
+  "version": "0.1.0",
+  "main": "src/js/index.js",
```

Comment:
can we add the description field here as well? now that it is an option in the cli?